src/components/Dropzone.svelte

```diff
@@ -32,6 +32,8 @@
   export let containerStyles = "";
   export let disableDefaultStyles = false;
   export let name = "";
+  export let inputRef;

```

Comment:
If exposing this var, we should make this var have a good name.
I think that `inputElement` is the best name because the type is `HTMLInputElement`.

```suggestion
  export let inputElement;
```